README.md

```diff
@@ -42,24 +44,21 @@ Run the sample
 
 - Get the `client_secret_NNNN.json`
   [OAuth client ID][oauth2-installed-app] JSON file for the application from
```

Comment:
Good call I should link to https://developers.google.com/assistant/sdk/prototype/getting-started-pi-python/config-dev-project-and-account instead